### PR TITLE
Make user aware that findCharactarPos returns top of the line position

### PR DESCRIPTION
**Pull request by dogunbound**
This doesn't need any testing, just documentation changes. I really do think a lot of this confusion may be avoided if we atleast mention this "top of the line" stuff

### CODE REVIEW

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

**Comment by dogunbound** [comment]
Linking https://github.com/SFML/SFML/issues/2988# to this

File: include/SFML/Graphics/Text.hpp [hunk 1]
```diff
@@ -372,8 +372,9 @@ class SFML_GRAPHICS_API Text : public Drawable, public Transformable
     ///
     /// This function computes the visual position of a character
     /// from its index in the string. The returned position is
-    /// in global coordinates (translation, rotation, scale and
-    /// origin are applied).
+    /// the global coordinates (translation, rotation, scale and
+    /// origin are applied) at the top of the character's text
+    /// line.
```
Comment (eXpl0it3r) [inline include/SFML/Graphics/Text.hpp:377]: I'd swap it around:

> ...position is at the top of the character's line in global coordinates...

Comment (eXpl0it3r) [reply include/SFML/Graphics/Text.hpp:377]: Is "character's text line" descriptive enough? If you hadn't had these discussions over the past few days, would you understand what it means?

Comment (dogunbound) [reply include/SFML/Graphics/Text.hpp:377]: I'm thinking about adding some information to the SFML-Website repo with like an image that we can use to define:
"top of the line"
"bottom of the line"
"line height"
"glyph bounds"

Comment (dogunbound) [reply include/SFML/Graphics/Text.hpp]: ascender height is very clear and googlable way of documenting what it's doing. I'm happy with this imho

**Pull request closed without merging**